internal/test/integration_testproj.go

```diff
@@ -283,6 +273,10 @@ func (p *IntegrationTestProject) makeRootTempDir() {
 		var err error
 		p.tempdir, err = ioutil.TempDir("", "gotest")
 		p.Must(err)
+
+		// Fox for OSX where the tempdir is a symlink:
+		p.tempdir, err = filepath.EvalSymlinks(p.tempdir)
+		p.Must(err)
```

Comment:
...for a hypothetical new OS/platform which also creates temp files as symlinks? :)

PS. Added now.